### DISCUSSION TITLE
Is there any way for Cypress to ignore third party tools/plugin errors thrown in console?

### DISCUSSION THREAD

**Discussion by andreummaestre** [cypress-io/cypress#22880] [General]
Hello,

We are facing a recurrent random issue during our tests we don't seem to be able to workaround.

We are using hotjar as a plugin in our app for surveys on client satisfaction and so on. 
The thing is, hotjar adds a small widget in the side of the app, which then loads a few things, does some requests and so on. 

One of them it's a hj.widget.pollData.persist_condition()....

If during our execution, the request is done before the widget has loaded, a unhanlded exception is thrown and our test fails because of it. 
We have tried using retryOnStatusCodeFailure, but it doesn't work on this issues.

My question is, is there a way for us to make Cypress ignore those plugins/widgets or errors?

**Comment by stavares843** [comment]
hello! 👋 

have you tried to add this on the /cypress/support/e2e.js file?


```
import './commands'

Cypress.on('uncaught:exception', (err, runnable) => {
  // returning false here prevents Cypress from
  // failing the test
  return false
})
```
